app/views/about/more.json.rabl

```diff
@@ -0,0 +1,10 @@
+node(:uri)                  { site_hostname }
+node(:title)                { Setting.site_title }
```

Comment:
They are ultimately both calling the same things ... the request was mostly to match what we do in the html version of about/more.

As a side note here, I assumed you are doing this for api consumption...? If that's the case, than we might want to either a) add a `link` describing where to find this to the html version of about/more, b) think about whether `about/more` is actually the best url for this endpoint (as opposed to something in the official api), c) maybe this should just expand the existing instance api endpoint?